### PR TITLE
Add ConfigReader test

### PR DESCRIPTION
**Pull request by aminlatifi**
Added ConfigReader test
Upgraded discovery dependency
Resolve L2B-2816

### CODE REVIEW

File: packages/backend/discovery/ConfigReader.test.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
+import { ChainId, ConfigReader } from '@l2beat/discovery'
+import { expect } from 'earl'
+import path from 'path'
+// import { ConfigReader } from './ConfigReader'
```
Comment (adamiak) [inline packages/backend/discovery/ConfigReader.test.ts:4]: Please remove comment.

**Review by adamiak — CHANGES_REQUESTED** [review]

**Review by adamiak — CHANGES_REQUESTED** [review]
I suggest renaming `ConfigReader.test.ts` to `verifyConfigs.test.ts` (same in `describe(`). This file is not a proper *unit test* for ConfigReader class, rather it's a validation that config.jsonc files can be parsed.

**Comment by aminlatifi** [comment]
> I suggest renaming `ConfigReader.test.ts` to `verifyConfigs.test.ts` (same in `describe(`). This file is not a proper _unit test_ for ConfigReader class, rather it's a validation that config.jsonc files can be parsed.

Done

File: packages/backend/discovery/verifyConfigs.test.ts [hunk 2]
```diff
@@ -0,0 +1,81 @@
+import { ChainId, ConfigReader } from '@l2beat/discovery'
+import { expect } from 'earl'
+import path from 'path'
+
+describe('Verify Discovery config files', () => {
+  // FIXME: This is a temporary hack to make the tests while configs are in backend
```
Comment (mateuszradomski) [inline packages/backend/discovery/verifyConfigs.test.ts:6]: I think we can remove this comment, since the context with which it was written is invalid now

**Review by mateuszradomski — CHANGES_REQUESTED** [review]

**Review by mateuszradomski — APPROVED** [review]

**Review by adamiak — APPROVED** [review]

**Pull request merged**